From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: MichaelSimons <msimons@microsoft.com>
Date: Mon, 22 Nov 2021 17:06:53 +0000
Subject: [PATCH] Disable RestoreUseStaticGraphEvaluation for source-build

---
 eng/Build.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eng/Build.props b/eng/Build.props
index 2fff7c5c2..f2afe8064 100644
--- a/eng/Build.props
+++ b/eng/Build.props
@@ -3,7 +3,7 @@
 <Project>
 
   <PropertyGroup>
-    <RestoreUseStaticGraphEvaluation>true</RestoreUseStaticGraphEvaluation>
+    <RestoreUseStaticGraphEvaluation Condition="'$(DotNetBuildFromSource)' != 'true'">true</RestoreUseStaticGraphEvaluation>
   </PropertyGroup>
 
   <ItemGroup>
