From 42ad0211e9d6c609cc5c1e1d19d7b17e0e275285 Mon Sep 17 00:00:00 2001
From: Michael Simons <msimons@microsoft.com>
Date: Mon, 13 Sep 2021 14:00:35 +0000
Subject: [PATCH] Add AddMissingTargetPathMetadata target to SFX

---
 .../targets/sharedfx.targets                             | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/src/Microsoft.DotNet.SharedFramework.Sdk/targets/sharedfx.targets b/src/Microsoft.DotNet.SharedFramework.Sdk/targets/sharedfx.targets
index 511214ae..724e0593 100644
--- a/src/Microsoft.DotNet.SharedFramework.Sdk/targets/sharedfx.targets
+++ b/src/Microsoft.DotNet.SharedFramework.Sdk/targets/sharedfx.targets
@@ -171,6 +171,15 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="AddMissingTargetPathMetadata" DependsOnTargets="GetFilesToPackage">
+    <ItemGroup>
+      <FilesToPackage Update="@(FilesToPackage->HasMetadata('DestinationSubDirectory'))"
+                      Condition="'%(FilesToPackage.TargetPath)' == ''">
+        <TargetPath>%(DestinationSubDirectory)/%(FileName).%(Extension)</TargetPath>
+      </FilesToPackage>
+    </ItemGroup>
+  </Target>
+
   <UsingTask TaskName="GenerateSharedFrameworkDepsFile" AssemblyFile="$(DotNetSharedFrameworkTaskFile)" />
   <Target Name="_GenerateSharedFrameworkDepsFile"
           DependsOnTargets="GetFilesToPackage"
-- 
2.29.2

