From e09da5670bfc8df488b65aee94de992a1c75a6a4 Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@microsoft.com>
Date: Mon, 14 Sep 2020 18:43:27 -0500
Subject: [PATCH] Add target rid to NetCoreAppHostRids

---
 src/redist/targets/GenerateBundledVersions.targets | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/src/redist/targets/GenerateBundledVersions.targets b/src/redist/targets/GenerateBundledVersions.targets
index 7c6f72967..3ee0284e1 100644
--- a/src/redist/targets/GenerateBundledVersions.targets
+++ b/src/redist/targets/GenerateBundledVersions.targets
@@ -99,6 +99,20 @@
       <NetCoreAppHostRids Include="@(Net50AppHostRids);osx-arm64" />
       <NetCoreRuntimePackRids Include="@(Net50RuntimePackRids);osx-arm64" />
 
+      <!--
+        In source-build, we build the current RID from source, which may be
+        non-portable and/or not an official RID. However, we can only use the
+        apphost pack, not the runtime pack, because:
+        - Apphost packs are distributed in the SDK.
+        - Runtime packs are not shipped with the SDK, only on NuGet.
+
+        Adding the ability to distribute and use source-built runtime packs is
+        tracked by: https://github.com/dotnet/source-build/issues/1215
+      -->
+      <NetCoreAppHostRids
+        Condition="'$(DotNetBuildFromSource)' == 'true'"
+        Include="$(ProductMonikerRid)" />
+
       <AspNetCore30RuntimePackRids Include="
         win-x64;
         win-x86;
-- 
2.27.0.windows.1

