From bdad8007363d3cd6409de7994de21b0f0670b90c Mon Sep 17 00:00:00 2001
From: Tom Deseyn <tom.deseyn@gmail.com>
Date: Wed, 19 Oct 2022 13:01:12 +0200
Subject: [PATCH] source-build: support building with mono runtime.

---
 eng/SourceBuild.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/eng/SourceBuild.props b/eng/SourceBuild.props
index 32108e1f..3dfbd5ef 100644
--- a/eng/SourceBuild.props
+++ b/eng/SourceBuild.props
@@ -55,6 +55,7 @@
 
     <PropertyGroup>
       <InnerBuildArgs>$(InnerBuildArgs) /p:SourceBuildRuntimeIdentifier=$(TargetRuntimeIdentifier)</InnerBuildArgs>
+      <InnerBuildArgs Condition="'$(SourceBuildUseMonoRuntime)' == 'true'">$(InnerBuildArgs) /p:CrossgenOutput=false</InnerBuildArgs>
     </PropertyGroup>
   </Target>
 
-- 
2.37.3

