From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Logan Bussell <loganbussell@microsoft.com>
Date: Mon, 15 Aug 2022 10:48:02 -0700
Subject: [PATCH] Exclude M.A.Components.Web.JS.npmproj during source build

---
 eng/Build.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/eng/Build.props b/eng/Build.props
index 6599b9d6f2..8fe5008c26 100644
--- a/eng/Build.props
+++ b/eng/Build.props
@@ -135,6 +135,7 @@
                         BuildInParallel="false"
                         RestoreInParallel="false"
                         Exclude="@(ProjectToExclude)" />
+        <ProjectToExclude Condition=" '$(DotNetBuildFromSource)' == 'true'" Include="$(RepoRoot)src\Components\Web.JS\Microsoft.AspNetCore.Components.Web.JS.npmproj" />
 
         <ProjectToBuild Condition=" '$(BuildNodeJS)' == 'true'" Include="@(NodeJsProjects)" Exclude="@(ProjectToExclude)" />
         <ProjectToExclude Condition=" '$(BuildNodeJS)' != 'true'" Include="@(NodeJsProjects)" />
