From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Viktor Hofer <viktor.hofer@microsoft.com>
Date: Thu, 2 May 2024 17:22:41 +0000
Subject: [PATCH] Avoid ambiguous call site

---
 .../Workspace/Core/CodeGeneration/CodeGenerationHelpers.cs      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeGeneration/CodeGenerationHelpers.cs b/src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeGeneration/CodeGenerationHelpers.cs
index 3839eb370eb..8d272db6b1b 100644
--- a/src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeGeneration/CodeGenerationHelpers.cs
+++ b/src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeGeneration/CodeGenerationHelpers.cs
@@ -71,7 +71,7 @@ internal static class CodeGenerationHelpers
                 break;
             }

-            name = string.Join(".", [.. names]);
+            name = string.Join(".", names);
         }
         else
         {
